src/bindings/html/service.js

```diff
@@ -9,14 +9,18 @@ export const L10n = {
   change: null
 };
 
-export function initViews(langs) {
+export function initViews(pretranslated) {
```

Comment:
Passing resolved `langs` to `initViews` helps avoid one additional `Promise.resolve`.  Why remove it?